examples/keras_spark_mnist.py

```diff
@@ -0,0 +1,124 @@
+from __future__ import print_function
```

Comment:
My worry with doing that is that users will associate it with TensorFlow estimators or similar, or think of this as a "specialized" API instead of the default (for most users).